rewrite-java/src/main/java/org/openrewrite/java/JavaPrinter.java

```diff
@@ -843,6 +843,16 @@ public J visitNewClass(NewClass newClass, PrintOutputCapture<P> p) {
         return newClass;
     }
 
+    @Override
+    public J visitNullableType(J.NullableType nt, PrintOutputCapture<P> p) {
+        beforeSyntax(nt, Space.Location.UNARY_PREFIX, p);
+        visit(nt.getTypeTree(), p);
+        visitSpace(nt.getPadding().getTypeTree().getAfter(), Space.Location.UNARY_OPERATOR, p);
```

Comment:
Then probably also a dedicated `NULLABLE_OPERATOR`.

Although, possibly we should instead throw an `UnsupportedOperationException` and require extending languages to implement 🤷

Reply:
updated to `Space.Location.NULLABLE_PREFIX`

Reply:
@traceyyoshima  what do you think about this, does JS and C# print it the same way? 
if the answer is yes, we probably want to have a shared print method here.

Reply:
C# shares the same syntax: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types#declaration-and-assignment.

A `?` in TS represents an [optional property](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#optional-properties), nullability is defined by using a union type.